src/sd.jl

```diff
@@ -24,17 +24,38 @@ julia> a = [ x 2x
 julia> b = [1 2
             2 4];
 
-julia> @SDconstraint(model, a ⪰ b)
-[x - 1, 2 x - 2, 2 x - 2, x - 4] ∈ MathOptInterface.PositiveSemidefiniteConeSquare(2)
+julia> cref = @SDconstraint(model, a ⪰ b)
+ x - 1    2 x - 2
+ 2 x - 2  x - 4   ∈ PSDCone()
```

Comment:
I'd argue for a format like:
```
[x - 1,    2 x - 2;
 2 x - 2,  x - 4 ]  ∈ PSDCone()
```
This is more immediately understandable because it uses existing Julia syntax. I had to spend a second to parse the output format above.

Reply:
We rely on Julia printing of matrices ro leverage its alignment of columns and its use of dots for too long matrices.
That makes it difficult to dectect change of columns so it's hard to add the commas